lang/php7/Makefile

```diff
@@ -21,6 +21,7 @@ PKG_HASH:=074093e9d7d21afedc5106904218a80a47b854abe368d2728ed22184c884893e
 PKG_FIXUP:=libtool autoreconf
 PKG_BUILD_PARALLEL:=1
 PKG_USE_MIPS16:=0
+PKG_BUILD_DEPENDS:=PACKAGE_php7-mod-imap:uw-imap
```

Comment:
NAK

Reply:
This line will go away when you make the suggested change below to the `BuildModule` invocation.

Reply:
I think too that there is currently no way to prevent this dependency this way as the uw-imap is not a dynmaic library we can link against.

Reply:
Sorry, then what is the product of `uw-imap`?

Reply:
It's only library .a file for static linking, plus headers... in other words, #5065 only installs this .a file and headers on build machine and the php module pulls it in...